source/includes/connect/replica-set.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+// Replace the placeholder with your connection string
```

Comment:
S: I don't think you need this comment instructing users to replace placeholders. Since this code example specifically demonstrates how to connect to the `sampleRS` replica set running on port `27017`, there isn't anything to replace

Reply:
good catch. this was added by Credal.